pkg/infra/grn/grn.go

```diff
@@ -2,17 +2,16 @@ package grn
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
-
-	"cuelang.org/go/pkg/strconv"
 )
 
 type GRN struct {
 	// TenantID contains the ID of the tenant (in hosted grafana) or
 	// organization (in other environments) the resource belongs to. This field
 	// may be omitted for global Grafana resources which are not associated with
 	// an organization.
-	TenantID int
+	TenantID int64
```

Comment:
OrgID is currently int64... so this makes things easier :)

Reply:
hm - isn't tenant identifier often a string? I am not sure how it looks like in our cloud

Reply:
In cloud it is an int (and you can not have orgs)

I agree string makes more sense, but given that we are stuck in int land this is a more useful place to start